### PR TITLE
Make message editable

### PR DESCRIPTION
**Pull request by jspayd**
Also replaces private/view.html with the more recent public/view.html and updates public (grunt generated/copied) files.
@stvstnfrd @caseylitton

### CODE REVIEW

**Comment by caesar2164** [comment]
@jspayd, could the additions of the css and js `.min` and `.min.map` files be split out into a separate commit?

**Comment by caesar2164** [comment]
hmm, are those files supposed to be added in the Gruntfile PR?

**Comment by dcadams** [comment]
Could you write a unit test for this?

**Comment by caseylitton** [comment]
@jspayd Per @dcadams comment, There should be a similar test in free text response.  One that checks the student view context for specific keys.

**Comment by caseylitton** [comment]
@caesar2164 It might be better to merge the two PRs into one.  First commit is fix the grunt file and add the missing min files.  Subsequent commits are the feature update.

https://github.com/Stanford-Online/xblock-qualtrics-survey/pull/6

**Comment by jspayd** [comment]
@caesar2164 Those files were generated by Grunt and get updated any time you make changes to the source files (in the private folder). I didn't actually make any changes to the JS and Less files. I think someone else had previously made changes to the xblock by modifying the public files (rather than modifying the private files and getting Grunt to minify them). That should all be fixed now, but I could move the .min files to the Gruntfile commit if that would make more sense (since it's really only view.html and qualtricssurvey.py that I directly modified here.)

**Comment by jspayd** [comment]
@dcadams Working on it now.

**Comment by jspayd** [comment]
I added tests. What's the consensus on what PR the `.min` and `.min.map` files should be in? @dcadams @caesar2164 @caseylitton

**Comment by dcadams** [comment]
Actually, I'd put them in their own commit in this PR.

**Comment by caesar2164** [comment]
@jspayd you did make changes to the JS and LESS, in the Grunt PR... :)
That's why I'm saying those changes should go there.

**Comment by jspayd** [comment]
The changes I made to the JS and LESS files in the Grunt PR were actually just copying them from public into private (since they weren't in private already), but I think the `.min` and `.min.map` files were changed by Grunt because of that. I'll move these changes to their own commit in the Grunt PR.

**Comment by stvstnfrd** [comment]
I'm tagged, but I defer to other reviewers here.

**Comment by caseylitton** [comment]
@jspayd @caesar2164 I tested the grunt files PR.  https://github.com/Stanford-Online/xblock-qualtrics-survey/pull/6

Those files(.min and .min.map) should be present in the other PR.  So they should not be newly added here.  

*edit, removed mistake.  The files are re-saved but not different unless they have specific changes.

File: qualtricssurvey/tests.py [hunk 1]
```diff
@@ -36,6 +36,8 @@ def test_student_view(self):
         self.assertIn(url_frag, fragment.content)
         url_frag = '>" target="_blank">Begin Survey'
         self.assertIn(url_frag, fragment.content)
+        message = '<p>' + xblock.message + '</p>'
+        self.assertIn(message, fragment.content)
```
Comment (caesar2164) [inline qualtricssurvey/tests.py]: There probably should be a specific test for the editable message. Like testing to make sure if you change the message it gets changed in the xBlock output?

**Review by caesar2164 — CHANGES_REQUESTED** [review]

**Comment by jspayd** [comment]
@caesar2164 @caseylitton I added the test @caesar2164 requested.

**Comment by caesar2164** [comment]
test this please

**Review by caesar2164 — APPROVED** [review]
nice!

**Comment by caseylitton** [comment]
+1,   Squash your commits and then add a revision commit.  

Do that similar to one Giulio just did for free text.  https://github.com/Stanford-Online/xblock-free-text-response/pull/20

**Comment by jspayd** [comment]
@caseylitton Done!

**Pull request merged**